lbry/tests/integration/test_claim_commands.py

```diff
@@ -617,7 +692,7 @@ class StreamCommands(ClaimTestCase):
         channels = await self.out(self.daemon.jsonrpc_channel_list(account1_id))
         self.assertEqual(len(channels), 1)
         self.assertEqual(channels[0]['name'], '@spam')
-        self.assertEqual(channels, await self.out(self.daemon.jsonrpc_channel_list()))
+        self.assertEqual(channels, await self.out(self.daemon.jsonrpc_channel_list(account1_id)))
```

Comment:
Does this test make sense to you? After the change it's just checking that `self.daemon.jsonrpc_channel_list(account1_id) == self.daemon.jsonrpc_channel_list(account1_id)`, is that helpful?